simplq/src/store/appSlice.js

```diff
@@ -14,49 +14,58 @@ const appSlice = createSlice({
   reducers: {
     setQueueName: (state, action) => {
       const queueName = action.payload;
+      // eslint-disable-next-line no-param-reassign
```

Comment:
I ended up disabling `no-param-reassign` for these lines. Looks like immerjs (the immutablility library redux toolkit uses behind the scenes) and this rule doesn't go well together, and this is the easiest fix I could come up with.

https://github.com/immerjs/immer/issues/189